.github/actions/spin-ci-dependencies/action.yml

```diff
@@ -129,3 +135,11 @@ runs:
       if: ${{ inputs.tinygo == 'true' }}
       with:
         version: ${{ inputs.tinygo-version }}
+
+    ## Install vcpgk
```

Comment:
nit: this is setting up openssl using vcpgk right?

Reply:
Ah yea, bad naming. Perhaps we do that as a follow up to not block this PR more?